README.md

```diff
@@ -1,284 +1,126 @@
-# `@cloudflare/next-on-pages`
+# ⚡▲ next-on-pages ▲⚡
 
-Reference:
+next-on-pages is a CLI tool that you can use to build and develop [Next.js](https://nextjs.org/) applications so that they can run on the [Cloudflare Pages](https://pages.cloudflare.com/) platform (and integrate with the various Cloudflare offerings such as KV stores, and Durable Objects).
```

Comment:
```suggestion
next-on-pages is a CLI tool that you can use to build and develop [Next.js](https://nextjs.org/) applications so that they can run on the [Cloudflare Pages](https://pages.cloudflare.com/) platform (and integrate with Cloudflare's various other product offerings such as KV, D1, R2 and Durable Objects).
```